app/src/ui/merge-branch/merge.tsx

```diff
@@ -38,6 +38,8 @@ interface IMergeProps {
    */
   readonly recentBranches: ReadonlyArray<Branch>
 
+  readonly preselectedBranch?: Branch
```

Comment:
I'm open to suggestions for a better name...

This prop is used to select a branch to merge in the merge dialog.